### ISSUE TITLE
Ability to transfer a project

### ISSUE THREAD

**Issue by cpottiers** [gmessner/gitlab4j-api#276]
Can you please add the ability to transfer a project to a specific namespace in ProjetApi ?

```
Introduced in GitLab 11.1.
PUT /projects/:id/transfer
Attribute : namespace | integer/string | required | The ID or path of the namespace to transfer to project to
```

**Comment by gmessner** [comment]
@cpottiers 
```ProjectApi.transferProject()``` has been added and released in 4.8.59.  Please verify and close.

**Comment by cpottiers** [comment]
:+1: 
works like a charm!
